.gitlab-ci.yml

```diff
@@ -69,3 +70,14 @@ ubuntu-python3-test:
     myconfig: "maxset"
     python_version: "3"
 
+status_success:
+  stage: status
+  image: ubuntu:xenial
+  script: bash maintainer/gh_post_status.sh success
+  when: on_success
+
+status_failure:
+  stage: status
+  image: ubuntu:xenial
+  script: bash maintainer/gh_post_status.sh failure
+  when: on_failure
```

Comment:
Needs
```
tags:
  - docker
```

Reply:
Why?

Reply:
Otherwise these jobs can end up on the Mac builders.

Reply:
The macOS machines don't accept jobs without tags

Reply:
Not in our current configuration. But if we ever have other non-Docker builders, we have a problem. Just make the dependency on Docker explicit by adding the tag.

Reply:
oh no, you think of windows build machines??? NOOOOOOO